### PR TITLE
two tower feature tests

### PR DESCRIPTION
**Pull request by hamham223**
### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

**Comment by hamham223** [comment]
@hkvision

File: python/friesian/example/wnd/recsys2021/wnd_preprocess_recsys.py [hunk 1]
```diff
@@ -226,9 +230,11 @@ def transform_label(tbl):
 
     user_index = train_tbl.gen_string_idx({'src_cols': ['engaged_with_user_id', 'enaging_user_id'],
                                            'col_name': 'user_id'})
+    item_index = train_tbl.gen_string_idx({'src_cols': ['tweet_id'],
+                                           'col_name': 'sim_tweet_id'})
```
Comment (hkvision) [inline python/friesian/example/wnd/recsys2021/wnd_preprocess_recsys.py]: no need this

File: python/friesian/example/wnd/recsys2021/wnd_preprocess_recsys.py [hunk 2]
```diff
@@ -169,6 +169,10 @@ def encode_user_id(tbl):
         .encode_string("user_id", user_index, broadcast=False) \
         .rename({"user_id": "enaging_user_id"})\
         .fillna(0, "enaging_user_id")
+    tbl = tbl.rename({"tweet_id": "sim_tweet_id"}) \
+        .encode_string("sim_tweet_id", item_index, broadcast=False) \
```
Comment (hkvision) [inline python/friesian/example/wnd/recsys2021/wnd_preprocess_recsys.py]: just encode string

**Comment by hamham223** [comment]
merged in intel-analytics:main, closing

**Pull request closed without merging**